<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PDFSettingsWidgetBase</class>
 <widget class="QWidget" name="PDFSettingsWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>431</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Enhance thin lines:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="kcfg_EnhanceThinLines">
       <item>
        <property name="text">
         <string>No</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Solid</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shape</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="kcfg_CheckOCSPServers">
     <property name="toolTip">
      <string>Enabling this option will allow Okular to contact 3rd-party OCSP servers to check if the certificates used for digital signing have been revoked since their creation.</string>
     </property>
     <property name="text">
      <string>Check revocation of digital signatures' certificates using 3rd-party servers</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="certDBGroupBox">
     <property name="title">
      <string>Certificate Database</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QRadioButton" name="kcfg_UseDefaultCertDB">
        <property name="text">
         <string>Default:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="customRadioButton">
        <property name="text">
         <string>Custom:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="defaultLabel">
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="certificatesGroup">
     <property name="title">
      <string>Available Certificates</string>
     </property>
     <layout class="QVBoxLayout" name="certificatesPlaceholder"/>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="loadSignaturesButton">
     <property name="text">
      <string>&amp;Load Signatures</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
